plugins.go

```diff
@@ -0,0 +1,185 @@
+package skipper
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"plugin"
+	"strings"
+
+	"github.com/zalando/skipper/filters"
+	"github.com/zalando/skipper/routing"
+)
+
+func findAndLoadPlugins(o *Options) error {
+	found := make(map[string]string)
+
+	for _, dir := range o.PluginDirs {
+		filepath.Walk(dir, func(path string, info os.FileInfo, err error) error {
+			if err != nil || info.IsDir() {
+				return nil
```

Comment:
maybe it would be worth to at least print a warning in case err != nil

Reply:
I would even suggest to do a log.Fatal, because if you roll this out with systems like Kubernetes, this would automatically stop the roll out. If you ignore errors, but the production system thinks to have the activated plugin, this would be a much bigger incident, then that the roll out is stopped.